PWGDQ/Core/CutsLibrary.cxx

```diff
@@ -2742,6 +2820,41 @@ AnalysisCut* o2::aod::dqcuts::GetAnalysisCut(const char* cutName)
     cut->AddCut(VarManager::kTPCnSigmaPr, 3.5, 999);
     return cut;
   }
+  //
+  // New test for jpsi2ee PID selection
+  //
+  if (!nameStr.compare("jpsi_TPCPID_TEST_1_Corr")) {
+    cut->AddCut(VarManager::kTPCnSigmaEl_Corr, -3.0, 3.0);
+    cut->AddCut(VarManager::kTPCnSigmaPi_Corr, 3.0, 999, false, VarManager::kP, 0.0, 2.0);
+    cut->AddCut(VarManager::kTPCnSigmaPi_Corr, 2.0, 999, false, VarManager::kP, 2.0, 999);
+    cut->AddCut(VarManager::kTPCnSigmaPr_Corr, 3.0, 999, false, VarManager::kP, 0.0, 2.0);
+    cut->AddCut(VarManager::kTPCnSigmaPr_Corr, 2.0, 999, false, VarManager::kP, 2.0, 999);
+  }
+  if (!nameStr.compare("jpsi_TPCPID_TEST_1")) {
+    cut->AddCut(VarManager::kTPCnSigmaEl, -3.0, 3.0);
+    cut->AddCut(VarManager::kTPCnSigmaPi, 3.0, 999, false, VarManager::kP, 0.0, 2.0);
+    cut->AddCut(VarManager::kTPCnSigmaPi, 2.0, 999, false, VarManager::kP, 2.0, 999);
+    cut->AddCut(VarManager::kTPCnSigmaPr, 3.0, 999, false, VarManager::kP, 0.0, 2.0);
+    cut->AddCut(VarManager::kTPCnSigmaPr, 2.0, 999, false, VarManager::kP, 2.0, 999);
+  }
+  if (!nameStr.compare("jpsi_TPCPID_TEST_2_Corr")) {
+    cut->AddCut(VarManager::kTPCnSigmaEl_Corr, -3.0, 3.0, false, VarManager::kP, 0.0, 2.0);
+    cut->AddCut(VarManager::kTPCnSigmaEl_Corr, -1.0, 4.0, false, VarManager::kP, 0.0, 2.0);
```

Comment:
did you intend to have this from 2 to 999 GeV?

Reply:
Yes but it was too optimisitic. I changed it to 4 to 999 instead